pkgs/applications/science/programming/fdr/default.nix

```diff
@@ -1,9 +1,10 @@
-{lib, stdenv, fetchurl, qtbase, qtx11extras, ncurses5, xorg, zlib, python27Packages}:
+{lib, stdenv, qtbase, qtx11extras, ncurses5, xorg, zlib, python27Packages}:
 stdenv.mkDerivation {
-  name = "fdr-4.2.3";
-  src = fetchurl {
-    url = "https://www.cs.ox.ac.uk/projects/fdr/downloads/fdr-3789-linux-x86_64.tar.gz";
-    sha256 = "0n2yqichym5xdawlgk3r7yha88k7ycnx6585jfrcm7043sls1i88";
+  pname = "fdr";
+  version = "4.2.7";
+  src = builtins.fetchTarball {
```

Comment:
That is not allowed in nixpkgs, see https://gist.github.com/GrahamcOfBorg/65512c14a5db03f480c30d41f8871e03. Please switch back to fetchurl.

Reply:
Ah, when I saw the forbidden in the evaluation I thought that might be the case. Have reverted that now.